### PR TITLE
Add analysis and plotting of prediction errors

### PR DESCRIPTION
**Pull request by dansbecker**

### CODE REVIEW

**Comment by dansbecker** [comment]
@sallamander I think this is pretty self-explanatory.  Let me know what you see in it.  If it looks good, I'd be interested to see the graphs from one of the models you've been running on AWS.

File: motion_tracker/model/eval_model.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import pandas as pd
+import seaborn as sns
+from matplotlib import pyplot as plt
+from motion_tracker.model.model import make_model
+from motion_tracker.utils.image_generator import AlovGenerator
+
+if __name__ == "__main__":
+    img_edge_size = 224
+    backend_id = 'th'
+    error_analysis_dir = 'work/error_analysis/'
```
Comment (sallamander) [inline motion_tracker/model/eval_model.py]: I think to have this line up with the below we should have './work/error_analysis'. Doesn't really matter, though.

File: motion_tracker/model/eval_model.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+        this_coord_pred = pred_df['pred_' + coord]
+        this_coord_actual = pred_df[coord]
+        pred_df[coord+'_error'] = (this_coord_actual - this_coord_pred).abs()
+        my_plot = sns.jointplot(this_coord_pred, this_coord_actual, kind="kde")
+        my_plot.savefig(error_analysis_dir + coord + '.png')
+    pred_df['mean_coordinate_error'] = pred_df.eval('(x0_error + y0_error + x1_error + y1_error) / 4')
+    error_plot = sns.distplot(pred_df.mean_coordinate_error)
+    plt.savefig(error_analysis_dir + 'mean_coordinate_error_dist.png')
+    plt.close()
+    pred_df.to_csv('prediction_error_info.csv', index=False)
```
Comment (sallamander) [inline motion_tracker/model/eval_model.py]: I think this path should also have the `error_analysis_dir` in it, no? Did you mean to save it to the main directory?

**Comment by dansbecker** [comment]
@sallamander Fixed the stuff you commented about.. will you take another look at the code.  Also, do you like the current graph with the best fit line better than the previous graphs?  I'm not sure which I like better.

**Comment by sallamander** [comment]
@dansbecker this is good right? Anything left that you wanted to add in?

**Comment by dansbecker** [comment]
I think it's good.

**Pull request merged**